gcal/calendar.go

```diff
@@ -25,7 +25,25 @@ func (c *client) DeleteCalendar(remoteUserID string, calID string) error {
 
 // GetCalendars returns a list of calendars
 func (c *client) GetCalendars(remoteUserID string) ([]*remote.Calendar, error) {
-	return nil, errors.New("gcal GetCalendars not implemented")
+	service, err := calendar.NewService(context.Background(), option.WithHTTPClient(c.httpClient))
+	if err != nil {
+		return nil, errors.Wrap(err, "gcal GetNotificationData, error creating service")
```

Comment:
```suggestion
		return nil, errors.Wrap(err, "gcal GetNotificationData, error creating service")
```
What is this `gcal GetNotificationData` for?